docker-compose.yml

```diff
@@ -69,6 +57,15 @@ services:
       options:
         max-size: "1M"
         max-file: "10"
+    networks:
+      - opencdms-surface
+    expose:
+      - 8000
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.routers.django.rule=Host(`$HOST_FQDN`)"
```

Comment:
define a HOST_FQDN in bash (for opencdms we set HOST_FQDN=api.opencdms.org, because api.opencdms.org is the domain we want the certificate to be issued for)

Reply:
Can we have these PR for dev branch instead of main? This way we can assess changes first in cloud server. Main branch is the one we use to push to Belize production server.

Reply:
Hi @fabiosato - agreed, these changes are intended for testing locally and in the cloud.

@faysal-ishtiaq could you raise a new PR to the dev branch?

Reply:
Hi @fabiosato - agreed, these changes are intended for testing locally and in the cloud.

@faysal-ishtiaq could you raise a new PR to the